examples/generic/cluster.yaml

```diff
@@ -100,11 +100,4 @@ spec:
           limits:
             cpu: 1
             memory: 1Gi
-        volumes:
```

Comment:
Is this related?

Reply:
Not, I can extract it to separate PR. It was added in #184 and it broke cluster deployment